File: typescript-angular/API/Client/Queue.ts
/// <reference path="api.d.ts" />

namespace API.Client {
    'use strict';

    export interface Queue {

        "id"?: string;

        "tableName"?: string;

        "createdAt"?: Date;

        "updatedAt"?: Date;

        "aCL"?: ACL;

        "type"?: string;

        "expires"?: Date;

        "queueType"?: string;

        "retry"?: string;

        "name"?: string;

        "subscribers"?: Array<string>;

        "messages"?: Array<QueueMessage>;
    }

}

File: akka-scala/src/main/scala/io/swagger/client/model/Queue.scala
package io.swagger.client.model

import io.swagger.client.core.ApiModel
import org.joda.time.DateTime


case class Queue (
  id: Option[String],
  tableName: Option[String],
  createdAt: Option[DateTime],
  updatedAt: Option[DateTime],
  aCL: Option[ACL],
  `type`: Option[String],
  expires: Option[DateTime],
  queueType: Option[String],
  retry: Option[String],
  name: Option[String],
  subscribers: Option[Seq[String]],
  messages: Option[Seq[QueueMessage]])
   extends ApiModel


